spec/controllers/api/v1/skills_controller_spec.rb

```diff
@@ -23,13 +28,13 @@
   end
 
   describe "create" do
+    let!(:group) { create(:group) }
 
     it "creates a skill (with full parameters)" do
-      group = create(:group)
       post :create, format: :json, skill: {
-                                            name: "created",
-                                            featured: true,
-                                            group: group.name
+                                             name: "created",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.